components/UserObserver.php

```diff
@@ -39,24 +39,25 @@ protected function getCSDClient(): CSDClient
      * @param array $params
 	 * @return User
      */
-	public function updateUser($params) :User
-	{		
-		/*if (in_array($params['username'],Yii::app()->params['local_users'])) {
+	public function updateUser($params)
+	{
+		if (in_array($params['username'],Yii::app()->params['local_users'])) {
 			return;
-		}*/
+		}
 
-		//if (Yii::app()->params['mehstaffdb_always_refresh'] || $this->isStale($params['username'])) {
-			//try {
-				$username = "WILLIAMSS";//$params['username'];
+		if (Yii::app()->params['mehstaffdb_always_refresh'] || $this->isStale($params['username'], $params['institution_authentication_id'])) {
+			try {
+				$username = $params['username'];
+				$institution_authentication_id = $params['institution_authentication_id'];
```

Comment:
Not necessarily a change request, but please be aware that the `institution_authentication_id` here is not equivalent to `institution_id`.